app/styles/node.less

```diff
@@ -0,0 +1,108 @@
+.node-detail {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.node-detail__row {
+  display: flex;
+  justify-content: space-between;
+  width: 50%;
+}
+
+.node-detail__cell {
+  width: 50%;
```

Comment:
what does it look like in small screen ?
if you want the elements of your flex container to have the same width, you can set flex-grow = 1